changelog.d/pa-3026.fixed

```diff
@@ -0,0 +1,4 @@
+For the `nonroot` Docker build stage, moved `semgrep-core` to
+`/home/semgrep/bin` and updated `$PATH` env variable with the
```

Comment:
Those changelog entries are directed to the users of semgrep; they don't really care about how things were achieved; What matters is more "You can now run semgrep --install-semgrep-pro in nonroot docker images".